keybot/main.go

```diff
@@ -12,9 +12,12 @@ import (
 	"strings"
 
 	"github.com/keybase/slackbot"
+	"github.com/keybase/slackbot/keybot/win_jenkins"
```

Comment:
Don't use underscore for package names in go... maybe just `jenkins` and maybe under slackbot instead of keybot..

e.g. `keybase/slackbot/jenkins`